dss/__init__.py

```diff
@@ -16,6 +16,8 @@
 # Constants common to the indexer and query route.
 DSS_ELASTICSEARCH_INDEX_NAME = "hca"
 DSS_ELASTICSEARCH_DOC_TYPE = "hca"
+DSS_ELASTICSEARCH_QUERIES_INDEX_NAME = "queries"
+DSS_ELASTICSEARCH_QUERIES_DOC_TYPE = "queries_type"
```

Comment:
I think it may be better to have separate values because we could accidentally use the wrong variable somewhere and not know until the value changes, which would break things.